pytest.ini

```
@@ -24,11 +24,6 @@ doctest_optionflags = ALLOW_UNICODE ELLIPSIS
 filterwarnings =
     error
 
```

Since the deprecation warning is happening in the third-party lib that we still support, we should ignore `DeprecationWarning` but only in that lib, not our own code base:
```suggestion
    # TODO: delete the following ignores once Ansible that we support gets rid of `imp`	
    # Ref: https://github.com/ansible/ansible-lint/pull/734
    ignore:the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses:DeprecationWarning:ansible.plugins.loader
```

---

(alternatively, it could be possible to not use that API from the third party that causes this, but only if it provides an alternate implementation)